packages/react-aria/src/hooks/useAriaButton.ts

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { ObjectShorthandProps, useEventCallback } from '@fluentui/react-utilities';
+
+export function useARIAButton(shorthand: ObjectShorthandProps<React.ButtonHTMLAttributes<HTMLElement>>) {
+  if (!shorthand.hasOwnProperty('as')) {
+    shorthand.as = 'button';
+  }
+  if (!shorthand.hasOwnProperty('children')) {
+    shorthand.children = null;
+  }
+
+  const { onClick, onKeyDown, onKeyUp, as, disabled } = shorthand;
+
+  const onClickHandler = useEventCallback((ev: React.MouseEvent<HTMLElement>) => {
+    if (disabled) {
+      ev.preventDefault();
```

Comment:
```suggestion
      ev.preventDefault();
      ev.stopPropagation();
```

You can double check but I think native buttons don't even propagate

Reply:
Just Double checking, https://codesandbox.io/s/muddy-water-eng1n?file=/src/App.tsx
Yeah... native buttons do propagate

Reply:
I meant for `disabled` but it seems you added it later